app/index.js

```diff
@@ -7,7 +7,7 @@
 
 var path = require('path');
 var SG = require('strong-globalize');
-SG.SetRootDir(path.join(__dirname, '..'));
+SG.SetRootDir(path.join(__dirname, '..'), {autonomousMsgLoading: 'all'});
```

Comment:
> Just the top modules. I will probably add it to loopback as well.
I'm just curious; is this needed for `loopback-datasource-juggler`, which is one of the main modules.